.github/workflows/ci.yml

```diff
@@ -29,9 +29,6 @@ jobs:
         with:
           submodules: true
 
-      - name: check bash
```

Comment:
Why is this check removed? It was checking something pretty subtle and weird but I see there are no comments as to what exactly :)

Reply:
(Not against removing, just checking why)

Reply:
I looked at the code and couldn't understand why this check needs to be present. It was doing no useful work (basically `if true; then echo ok; fi` then repeat for another time), and there was no comment to explain its significance. I thought it was legacy leftovers not cleaned up. I double checked my CI and packaging runs and this step never failed (I would be surprised if it did). My guess is it might be checking whether the Windows association step worked (script is run with bash rather than powershell), and was likely added for debugging purposes, so I removed it. If anything breaks in the future I can always come back to it.
Please do share any extra context you have on what the problem was.